### PR TITLE
Dotty 0.3.0-RC2 announcement

### PR DESCRIPTION
**Pull request by allanrenucci**

### CODE REVIEW

**Comment by allanrenucci** [comment]
@heathermiller Can you please review?

**Comment by heathermiller** [comment]
LGTM! @allanrenucci do you want to merge now, or wait till tomorrow?

**Comment by allanrenucci** [comment]
The release has not landed into maven yet. We will wait until tomorrow.

**Comment by heathermiller** [comment]
Sounds good. If that's the case, then please update the date on this article by renaming the file 😉

**Comment by smarter** [comment]
Minor suggestions:
- https://github.com/lampepfl/dotty/releases/tag/0.3.0-RC1 includes a list of all contributors who participated in the release, I think this is a nice thing to put in the blog post, instead of the subset currently listed.
- It would be nice to add a small section on getting started contributing to Dotty which points people to http://scala-lang.org/blog/2016/10/14/dotty-errors.html and https://github.com/lampepfl/dotty/issues?q=is%3Aissue+is%3Aopen+label%3Aexp%3Anovice

**Comment by allanrenucci** [comment]
@smarter I have added a `Reporting Bugs` and `Contributing` sections

**Comment by allanrenucci** [comment]
@heathermiller Can you merge it? Thanks!

**Comment by heathermiller** [comment]
Done 🙂
Should be live in a minute or two!

**Comment by allanrenucci** [comment]
There seems to be an issue with the rendering of the [website](https://www.scala-lang.org/blog/2017/09/07/third-dotty-milestone-release.html)

File: blog/_posts/2017-09-07-third-dotty-milestone-release.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+---
+layout: blog
```
Comment (ashawley) [inline blog/_posts/2017-09-07-third-dotty-milestone-release.md]: Layout should be `blog-detail`, not `blog`.

Comment (heathermiller) [reply blog/_posts/2017-09-07-third-dotty-milestone-release.md]: 👍

**Comment by heathermiller** [comment]
In general @allanrenucci, you need to follow the instructions in the README to build and test the blog article locally. It isn't enough to just make a PR with some markdown in it to scala-lang.

**Comment by heathermiller** [comment]
Thanks @ashawley for debugging it so quickly!

**Comment by heathermiller** [comment]
Fixed in 165564aae2c579d69bfbf7f5e8204821cc31257e

**Comment by smarter** [comment]
@heathermiller Not sure whom to ask for this, but could the blog post be tweeted by https://twitter.com/scala_lang ?

**Comment by heathermiller** [comment]
Sure, I'll do it now.

**Comment by heathermiller** [comment]
Done: https://twitter.com/scala_lang/status/906290538214813702

**Pull request merged**